README.md

```diff
@@ -1,15 +1,101 @@
 # getappmap/archive-appmap
 
+> To get started with AppMap in GitHub actions, you need to start by installing the [AppMap App on the official GitHub Marketplace](https://github.com/marketplace/get-appmap)
+
+> To see a step-by-step example of how to install this action into your software project, [review the official AppMap Documentation](http://appmap.io/docs/analysis/in-github-actions).
+
 GitHub action to build and store an AppMap archive.
```

Comment:
It's used during initial setup to create a `Configuration report`.

It's used in matrix builds to create a archive of AppMaps on a worker node and save them to the GitHub cache.

It's used when a PR is merged (both in matrix and non-matrix mode) to build and store a baseline artifact. This is also done on a schedule (cron) to ensure that the artifacts don't expire and disappear. Default artifact expiration is 90 days afaik. This is a configurable GitHub org or project setting.